package.json

```diff
@@ -5,24 +5,24 @@
   "license": "MIT",
   "main": "./src/index.js",
   "devDependencies": {
-    "@babel/core": "^7.20.5",
-    "@babel/preset-env": "^7.20.2",
-    "autoprefixer": "^10.4.13",
-    "babel-loader": "^9.1.0",
-    "css-loader": "^7.1.0",
-    "mini-css-extract-plugin": "^2.7.1",
+    "@babel/core": "^7.26.0",
+    "@babel/preset-env": "^7.26.0",
+    "autoprefixer": "^10.4.20",
+    "babel-loader": "^9.2.1",
+    "css-loader": "^7.1.2",
+    "mini-css-extract-plugin": "^2.9.2",
     "path": "^0.12.7",
-    "postcss": "^8.4.20",
-    "postcss-loader": "^8.0.0",
-    "sass": "^1.56.1",
-    "sass-loader": "^14.1.1",
+    "postcss": "^8.4.49",
+    "postcss-loader": "^8.1.1",
+    "sass": "^1.83.0",
+    "sass-loader": "^16.0.4",
```

Comment:
from v16, sass-loader [uses the "modern Sass JS Api"](https://github.com/webpack-contrib/sass-loader/releases/tag/v16.0.0). As we're importing some Scss files with the addon-client-builder, idk if this affects the build behavior. However adapting the webpack config seems to be rather straightforward, as [you did](https://github.com/demos-europe/demosplan-core/pull/4095) [this before](https://github.com/demos-europe/demosplan-core/pull/4220) :)